lib/ui/components/cv_social_card.dart

```diff
@@ -1,26 +1,34 @@
 import 'package:flutter/material.dart';
 import 'package:mobile_app/cv_theme.dart';
 import 'package:mobile_app/utils/url_launcher.dart';
+import 'package:url_launcher/url_launcher.dart';
 
 class CircuitVerseSocialCard extends StatelessWidget {
   const CircuitVerseSocialCard({
     required this.imagePath,
     required this.title,
     required this.description,
     required this.url,
+    required this.choice,
     Key? key,
   }) : super(key: key);
 
   final String imagePath;
   final String title;
   final String description;
   final String url;
+  final int choice;
 
   @override
   Widget build(BuildContext context) {
     return GestureDetector(
       onTap: () async {
-        launchURL(url);
+        if (choice == 1) {
+          final url = Uri.parse('mailto:support@circuitverse.org');
+          await launchUrl(url);
+        } else {
+          launchURL(url);
+        }
```

Comment:
You can consider to manage this from the function itself rather than overriding it here and adding new argument.
You can override it in the lib/utils/url_launcher.dart , launchURL function , to check whether the url starts with mailto, if match you can parse the uri and call required function